test/src/com/xilinx/rapidwright/design/TestDCPLoad.java

```diff
@@ -125,6 +127,13 @@ public void testDCPFromVivado2022_1(String dcp) {
     @ParameterizedTest
     @ValueSource(strings = {"picoblaze_2022.2.dcp"})
     public void testDCPFromVivado2022_2(String dcp) {
-        RapidWrightDCP.loadDCP(dcp);
+        Design design = RapidWrightDCP.loadDCP(dcp);
+        for (Cell c : design.getCells()) {
+            if (c.getName().contains("processor")) {
```

Comment:
Are you restricting it to `processor` for runtime reasons, or is there a reason this check would fail for non-`processor`-containing paths?

Reply:
This is an issue with hierarchy.  There are some `Cell`s that should be excluded at the top level (`PAD` or port cells) that don't have a hierarchical equivalent. All cells with hierarchy contain `processor` so this was an easy way to satisfy the scenario.